web/modules/custom/ai_screening_project_track/src/ProjectTrackInterface.php

```diff
@@ -18,6 +18,16 @@ interface ProjectTrackInterface extends ContentEntityInterface, EntityChangedInt
    */
   public function getType(): String;
 
+  /**
+   * Get title.
+   */
+  public function getTitle(): String;
```

Comment:
Using `String` (with an uppercase `S`) is a typo introduced by me in https://github.com/itk-dev/ai-screening/pull/18. It should be `string`.